google/cloud/spanner_v1/database.py

```diff
@@ -726,6 +730,15 @@ def execute_pdml():
 
                 return result_set.stats.row_count_lower_bound
 
+            with SessionCheckout(self._pool) as session:
```

Comment:
why is the trace_call started after SessionCheckout?
Since session checkout is part of this method, we should start the span before this with block itself.
Then only session related operations such as getting session etc will be part of pdml parent span.